### PR TITLE
[DO NOT MERGE] SEO edits for top scoring industry tests

### PR DESCRIPTION
**Pull request by jdmartinez36**
Edited title and description for length and keywords.

@DaniHalfin 

Edits from PR 2155.

### CODE REVIEW

File: microsoft-365/security/mtp/top-scoring-industry-tests.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Top scoring in industry tests (AV-TEST, AV Comparatives, SE Labs, MITRE ATT&CK)
+title: Microsoft Threat Protection - Top scoring in industry tests
```
Comment (levinec) [inline microsoft-365/security/mtp/top-scoring-industry-tests.md:2]: Shouldn't it be the title, and then the product? "Top scoring in industry tests - Microsoft Threat Protection"

File: microsoft-365/security/mtp/top-scoring-industry-tests.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 ms.reviewer: 
-description: Microsoft Defender ATP consistently achieves high scores in independent tests. View the latest scores and analysis.
+description: Learn why Microsoft Defender ATP consistently achieves high scores in independent av tests (AV-TEST, AV Comparatives, SE Labs, MITRE ATT&CK). View the latest scores and analysis.
```
Comment (levinec) [inline microsoft-365/security/mtp/top-scoring-industry-tests.md:4]: Microsoft Defender ATP should actually be updated to Microsoft Threat Protection. And it's not just AV tests anymore.

"Learn why Microsoft Threat Protection consistently achieves high scores in independent tests (AV-TEST, AV Comparatives, SE Labs, MITRE ATT&CK). View the latest scores and analysis.

**Comment by levinec** [comment]
Left some comments @jdmartinez36 and @DaniHalfin

**Comment by tuco-ulf** [comment]
@DaniHalfin is this ready to merge?

**Comment by levinec** [comment]
@tuco-ulf It doesn't look like the changes I commented about were made.

**Comment by DaniHalfin** [comment]
Addressed separately

**Pull request closed without merging**